index.js

```diff
@@ -23,7 +23,6 @@ function format(f, args, opts) {
   }
   var argLen = args.length
   if (argLen === 0) return f
-  var x = ''
```

Comment:
why was this removed?

Reply:
This variable is no longer used in the code. Seams that it has been used in previous versions.